Use sot-dynamic-pinocchio. Change of package name from sot-dynamic

--- CMakeLists.txt	2016-04-14 14:41:27.000000000 +0200
+++ CMakeLists.txt	2017-02-02 13:39:57.477133197 +0100
@@ -48,7 +48,7 @@
 
 ADD_REQUIRED_DEPENDENCY("dynamic-graph >= 3.0.0")
 ADD_REQUIRED_DEPENDENCY("sot-core >= 3.0.0")
-ADD_REQUIRED_DEPENDENCY("sot-dynamic >= 3.1")
+ADD_REQUIRED_DEPENDENCY("sot-dynamic-pinocchio >= 3.1")
 ADD_REQUIRED_DEPENDENCY("dynamic-graph-python >= 3.0.0")
 
 ADD_REQUIRED_DEPENDENCY("dynamic_graph_bridge_msgs")
